richeditor-compose/src/commonMain/kotlin/com/mohamedrejeb/richeditor/model/RichParagraph.kt

```diff
@@ -11,7 +11,7 @@ import com.mohamedrejeb.richeditor.ui.test.getRichTextStyleTreeRepresentation
 import com.mohamedrejeb.richeditor.utils.fastForEach
 import com.mohamedrejeb.richeditor.utils.fastForEachIndexed
 
-internal class RichParagraph(
+class RichParagraph(
```

Comment:
RichParagraph needs to be made public, or the model could be separated to core and taken as an implementation dependency , which will allow to hide from public API and also separating HTML and Markdown to their own modules